File: MyDiary/app/src/main/java/com/example/masafumi_ito/mydiary/MyApplication.java
package com.example.masafumi_ito.mydiary;

import android.app.Application;

import io.realm.Realm;

/**
 * Created by masafumi_ito on 2017/08/27.
 */
public class MyApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        Realm.init(this);
    }
}

File: MyScheduler/app/src/main/java/com/example/masafumi_ito/myscheduler/MyApplication.kt
package com.example.masafumi_ito.myscheduler

import android.app.Application

import io.realm.Realm
import io.realm.RealmConfiguration



class MyApplication : Application() {

    override fun onCreate() {
        super.onCreate()

        // Realmの初期化
        Realm.init(this)
        Realm.setDefaultConfiguration(RealmConfiguration.Builder().build())
    }
}
